WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -39,6 +41,14 @@ class LoginPrologueViewController: LoginViewController {
     ///
     private var prologueFlowTracked = false
 
+    /// Returns `true` is a configurable OR divider is needed for the list of prologue buttons
+    ///
+    /// `NUXStackedButtonsViewController` is used for creating buttons when this is `true`
+    ///
+    private var orDividerNeededInPrologueButtons: Bool {
+        configuration.enableWPComLoginOnlyInPrologue || configuration.enableSiteCreation
```

Comment:
I'm confused, should it be that the divider is needed when we enable WP.com & site credential logins and also site creation?

```suggestion
        configuration.enableWPComLoginOnlyInPrologue == false && configuration.enableSiteCreation
```

Reply:
Sorry for the confusion. This is actually for deciding whether to use the `NUXStackedButtonsViewController` or not. 

Renamed in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/695/commits/d92b4b546494c201567e55509295c8c8b868697f and updated comment in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/695/commits/c8aec5c9054f4570d137e4e4416694943b2ae1bd